controllers/user.js

```diff
@@ -36,7 +42,8 @@ export default {
       updatedAt: user.updatedAt,
       isActive: user.isActive,
       isAdmin: user.isAdmin,
-      isSuperAdmin: user.isSuperAdmin
+      isSuperAdmin: user.isSuperAdmin,
+      imageURL: user.imageURL
```

Comment:
I don't think you need to sign the token with the imageURL details. All the user data currently present in the token are to check **if a user is the owner of a resource before requesting the resource** or **if a user is currently active** or **if a user is admin/superadmin**. The imageURL won't really serve much here especially since it is allowed to be null.

Reply:
To this end, you could transfer this imageURL to the login response, so that we can have access to the imageURL when a user logs in too.

Reply:
The imageURL details is not added to the token from my end. It is only added to the responseObject.

Reply:
Oh, awesome then.